controllers/cards.js

```diff
@@ -1,8 +1,7 @@
-/* eslint-disable no-underscore-dangle */
-/* eslint-disable consistent-return */
-/* eslint-disable import/no-dynamic-require */
 const path = require('path');
+const NotFoundError = require('../errors/notFoundError');
 
+// eslint-disable-next-line import/no-dynamic-require
 const Card = require(path.join(__dirname, '../models/card'));
```

Comment:
Можно лучше: не использовать динамическое подключение.
Когда не требуется динамическое подключение модулей достаточно указывать относительный путь до файла. Либо, если это NPM-пакет, то указать путь начиная с названия NPM-пакета. Функция `require()` сама способна разрулить путь до файла:
```suggestion
const Card = require('../models/card');
```

После изменения нужно не забыть удалить подключение модуля `path`, т.к. он не используется - иначе будет ругаться линтер.